reana_client/cli/workflow.py

```diff
@@ -1458,3 +1464,84 @@ def workflow_close_interactive_session(workflow, access_token):  # noqa: D301
             sys.exit(1)
     else:
         display_message("Cannot find workflow {} ".format(workflow), msg_type="error")
+
+
+@workflow_sharing_group.command("share-add")
+@check_connection
+@add_workflow_option
+@add_access_token_options
+@click.option(
+    "-u",
+    "--user",
+    "users",
+    multiple=True,
+    help="Users to share the workflow with.",
+    required=True,
+)
+@click.option(
+    "-m",
+    "--message",
+    help="Optional message that is sent to the user(s) with the sharing invitation.",
+)
+@click.option(
+    "-v",
```

Comment:
The shorthand option `-v` is normally used for `--verbose` in other scripts, so I think we should either change it to another letter or even drop the shorthand version. What do you think?

Reply:
Changed it to allow for `--valid-until` only, removed the `-v` shorthand.

The changes were applied to the following superseding PR: #692